wagtail/embeds/models.py

```diff
@@ -22,22 +23,23 @@ class Embed(models.Model):
     If an instance of this model is deleted, it will be automatically refetched
     next time the embed code is needed.
     """
-    url = models.URLField()
+
+    url = models.TextField(validators=[URLValidator()])
```

Comment:
I know this sounds counterintuitive, but we actually don't ever invoke field or model-level validation for embeds, so we can safely drop the validators (for both fields)